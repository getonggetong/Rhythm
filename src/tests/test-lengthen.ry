init() {
	print(A4 * 2);
	print(A4.16 * 2);
	print(A4.2 * 2);
	print(A4.16 * 16);
	print([A4,B4,C4] * 2);
	print([Ab4.8,B4.8,C#4.8] * 2);
}