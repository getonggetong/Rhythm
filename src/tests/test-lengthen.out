A4.2
A4.8
A4.1
A4.1
[A4.2,B4.2,C4.2]
[Ab4.4,B4.4,C#4.4]